adicionando questao A (#1)
BRUTEUdesc/brute_class: contests/2020.1/rinha_de_calouros/prova/A/A.tex

\section{Problema A: Mesada Par}
\vspace{-0.52cm}
\noindent \begin{verbatim}Arquivo: A.[c|cpp|java|py]                           Tempo limite: 1 s
\end{verbatim}

> Omar é um garoto muito esperto e generoso da cidade EntreVila. Como ele sempre tem boas notas e é obediente, recebe mesada de seus pais. Por ser generoso, ele não fica com toda a mesada pra ele. Omar, as vezes, pega uma parte e doa para uma instituição de caridade da cidade. Mas, por outro lado, ele não é tão generoso assim, pois a parte que ele doa é a parte que o incomoda na mesada, ele só gosta de números pares. Se a mesada não é um valor par, ele doa a menor parte que ele pode tal que a mesada fique par, se a mesada é par ele não doa nada e guarda tudo.\\

Reviewer: ```suggestion
Omar é um garoto muito esperto e generoso da cidade EntreVila. Como ele sempre tem boas notas e é obediente, recebe mesada de seus pais. Por ser generoso, ele não fica com toda a mesada pra ele e doa para uma instituição de caridade da cidade. Porém ele não é tão generoso assim já que a ele doa varia de acordo com uma condição: ele só gosta de números pares. \\
Se a mesada não é um valor par, ele doa a menor parte que ele pode tal que a mesada fique par, já se a mesada é par ele não doa nada e guarda tudo em sua poupança. \\
```

> Omar mora com os pais e nunca sai de casa, portanto não tem gastos, guarda toda a mesada(fora a parte doada). Ele anotou as mesadas que seus pais o deram no ano passado e quer saber quanto guardou, no total, neste tempo. Como os números podem variar bastante, ele pediu sua ajuda. Ele pediu para você escrever um programa que, de acordo com as mesadas do ano, diga quanto guardou. Lembre que ele sempre coloca valores pares em seu cofre.\\

Reviewer: ```suggestion
Omar mora com os pais e nunca sai de casa, portanto não tem gastos e guarda tudo que sobra da mesada. \\
Em seu caderninho ele anotou as mesadas que seus pais o deram no ano passado mas esqueceu de anotar quanto ele realmente guardou. Como as habilidades matemáticas dele se limitam a saber se um número é par ou não, ele pediu para você escrever um programa que de acordo com as mesadas do ano diga quanto conseguiu acumular. \\
```

> A mesada de Omar não é pré-determinada nem tem um padrão necessariamente, segundo seus pais ela pode variar de 1 até 10^{9} reais.

Reviewer: Eu removeria toda essa frase

\subsection*{Entrada}

> A entrada consiste em 12 linhas cada uma com um inteiro $N$ representando a mesada daquele mês.

Reviewer: ```suggestion
A entrada consiste de um caso único de teste. O caso de teste contém 12 linhas, cada uma com um inteiro $N$ representando o valor da mesada recebido de seus pais naquele mês.
```
